devicon.json

```
@@ -1125,6 +1125,28 @@
         "color": "#343434",
         "aliases": []
     },
+    {
+        "name": "clarity",
+        "altnames": [],
```

You can add clarity-lang to altnames, since it's what they call it on github and their website. 
```suggestion
        "altnames": ["clarity-lang"],
```
Not sure if we should add `clarity-lang`, `clarity_lang`, `claritylang` or all three of them though. `clarity-lang` is the most used one, but the others also make it more searchable.

What do you think?

---

- `clarity_lang` seems to be used only on Twitter, because of username symbols restriction: alphanumeric and underscore only.
- `claritylang` on Google search will be treated like a mistake, even when showing the same results as `clarity-lang`. And unlike `vyperlang` case, they chose to keep the hyphen.
- `clarity-lang` seems to be the right one to me for the reasons written before and because searching will be assertive to their website.

If everything is ok I'll be committing your changes.

---

Good observations, sounds good to me! 👍